diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter2/kdoc/KdocComments.kt

```diff
@@ -106,18 +107,12 @@ class KdocComments(configRules: List<RulesConfig>) : DiktatRule(
             kdocBeforeClass?.let {
                 checkKdocBeforeClass(node, kdocBeforeClass, prevComment)
             }
-                ?: run {
```

Comment:
Yes, it would make sense to implement a rule, that would detect useless scope functions. Feel free to open an issue, we can address it after diktat 1.0.0

Reply:
https://github.com/cqfn/diKTat/issues/972